spec/spec_helper_acceptance.rb

```diff
@@ -1,315 +1,8 @@
-require 'beaker-rspec'
-require 'net/http'
-require 'pry'
-require 'securerandom'
-require 'yaml'
+# frozen_string_literal: true
 
-# Collect global options from the environment.
-if ENV['LOGSTASH_VERSION'].nil?
-  raise 'Please set the LOGSTASH_VERSION environment variable.'
-end
-LS_VERSION = ENV['LOGSTASH_VERSION']
-PUPPET_VERSION = ENV['PUPPET_VERSION'] || '4.10.7'
+# Managed by modulesync - DO NOT EDIT
+# https://voxpupuli.org/docs/updating-files-managed-with-modulesync/
 
-PE_VERSION = ENV['BEAKER_PE_VER'] || ENV['PE_VERSION'] || '3.8.3'
-PE_DIR = ENV['BEAKER_PE_DIR']
+require 'voxpupuli/acceptance/spec_helper_acceptance'
 
-if LS_VERSION =~ /(alpha|beta|rc)/
-  IS_PRERELEASE = true
-else
-  IS_PRERELEASE = false
-end
-
-def agent_version_for_puppet_version(puppet_version)
-  # REF: https://docs.puppet.com/puppet/latest/reference/about_agent.html
-  version_map = {
-    # Puppet => Agent
-    '4.9.4' => '1.9.3',
-    '4.8.2' => '1.8.3',
-    '4.8.1' => '1.8.2',
-    '4.8.0' => '1.8.0',
-    '4.7.1' => '1.7.2',
-    '4.7.0' => '1.7.1',
-    '4.6.2' => '1.6.2',
-    '4.6.1' => '1.6.1',
-    '4.6.0' => '1.6.0',
-    '4.5.3' => '1.5.3',
-    '4.4.2' => '1.4.2',
-    '4.4.1' => '1.4.1',
-    '4.4.0' => '1.4.0',
-    '4.3.2' => '1.3.6',
-    '4.3.1' => '1.3.2',
-    '4.3.0' => '1.3.0',
-    '4.2.3' => '1.2.7',
-    '4.2.2' => '1.2.6',
-    '4.2.1' => '1.2.2',
-    '4.2.0' => '1.2.1',
-    '4.1.0' => '1.1.1',
-    '4.0.0' => '1.0.1'
-  }
-  version_map[puppet_version]
-end
-
-def apply_manifest_fixture(manifest_name)
-  manifest = File.read("./spec/fixtures/manifests/#{manifest_name}.pp")
-  apply_manifest(manifest, catch_failures: true)
-end
-
-def expect_no_change_from_manifest(manifest)
-  expect(apply_manifest(manifest).exit_code).to eq(0)
-end
```

Comment:
I'd expect these to still be needed. Possibly others as well. Should they be moved into `spec/support/acceptance`?

Reply:
done